tsconfig.json

```diff
@@ -7,17 +7,12 @@
     "noImplicitAny": false, // TODO: Remove when all source are migrated to TS
```

Comment:
Are we here yet? Or no?

Reply:
Soon. I think we might have to migrate the tests.